docker.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+
+tlmgr option repository http://ftp.math.utah.edu/pub/tex/historic/systems/texlive/2019/tlnet-final
+tlmgr --no-persistent-downloads install sourcecodepro sourcesanspro sourceserifpro crimsonpro
+cd /data
+pandoc -f markdown -t latex docs/BR.md --table-of-contents -s --metadata-file assets/BR.yaml --pdf-engine=xelatex --template=templates/BR.latex -o docs/BR.pdf
```

Comment:
Why not invoke the makefile here? Why the repetition? Or have I overlooked and these are distinct?

Reply:
Oh well, see, there you go again, you and your "logic" and "reason". :-D That's a good plan; I'll finish figuring out fixing up the Makefiles and then just call it from here.

Reply:
Answered this above. The docker image lacks make.